### ISSUE TITLE
Unskip shouldn't autoskip the segment again if a smaller segment inside it is skipped

### ISSUE THREAD

**Issue by ajayyy** [ajayyy/SponsorBlock#1279]
Segment 1:00 to 2:00
Segment 1:10 to 1:20

unskip segment 2, current behavior will ski from 1:10 to 2:00

**Comment by LofhJann** [comment]
I disagree. 

If there are overlapping segments, in my opinion the only behaviour that makes sense is to skip longer segment. If it worked any other way, we would be only partially skipping the longer segment and it would not make any sense. In this case, there would be no way to actually skip the "segment 1". I understand there could be some use cases for proposed behaviour, but neither is good solution for this in my opinion. In either case there would be confusion why it works as it does.

The overhaul of whole manual skip system with options for how to handle overlapping segments would be probably only way to solve this "right way". 

Options could include for example:
1. Always skip overlapping segments (Would skip from 1:00 to 2:00)
2. Don't skip overlapping segments (Would skip from 1:00 -> 1:10, 1:10 -> 1:20, 1:20 -> 2:00)
3. Ask before skipping (Dropdown, what segments you want to skip)
4. Always skip selected types of segment (multi-choice, with selection of what types of segments you want to skip. If overlapping, ask user)

**Comment by mchangrh** [comment]
I disagree, segments should not in normal circumstances overlap more than 80%. If viewing normally and autoskipping, it would skip to the end of the larger segment, this is only in cases where the user is explicitly unskipping the larger segement to inspect the smaller one.
